libs/core/src/lib/grid-list/helpers/parse-layout-pattern.ts

```diff
@@ -0,0 +1,41 @@
+export function parseLayoutPattern(pattern: string): string {
+    const [xl, lg, md, s] = pattern.split('-');
```

Comment:
It would be great if we can apply only part of custom pattern, for example 'XL6-S12'. Other cases should apply from default.